build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -55,7 +55,7 @@ public void PublishArchive(string archiveFile, string channel, string version)
         public void PublishFile(string blob, string file)
         {
             CloudBlockBlob blockBlob = _blobContainer.GetBlockBlobReference(blob);
-            blockBlob.UploadFromFileAsync(file, FileMode.Open).Wait();
+            blockBlob.UploadFromFileAsync(file).Wait();
```

Comment:
Yes, the `FileMode` param was removed and hard-coded to `Open`. I couldn't find anything official mentioning it, so I checked the IL and confirmed that other than that, it's identical.

I'm not aware of any known-good way to do a private build on the master branch right now, so I don't think it's worth it. It seems a lot more reasonable to make the change and revert if needed.